core/domain/translation_services.py

```diff
@@ -226,8 +223,7 @@ def get_displayable_translation_languages(
 
 def get_translation_counts(
```

Comment:
Should we add a test for this?

Reply:
This is an old code, so I'm assuming there must exist a test for the same. I've just modified a line in this function.

Reply:
Gotcha, thanks.